### PR TITLE
nydus: support more compression algorithms

### PR DESCRIPTION
**Pull request by hsiangkao**
Apart from LZ4, let's add Zstd support too.

Signed-off-by: Gao Xiang \<hsiangkao@linux.alibaba.com\>

### CODE REVIEW

**Comment by hsiangkao** [comment]
note that I didn't run lint test since I have no such tool.

File: misc/config/config.yaml.nydus.tmpl [hunk 1]
```diff
@@ -46,6 +46,8 @@ converter:
       builder: nydus-image
       # specify nydus format version, possible values: `5`, `6` (EROFS-compatible), default is `5`
       rafs_version: 5
+      # specify nydus compression algorithm, `lz4block` by default
```
Comment (imeoer) [inline misc/config/config.yaml.nydus.tmpl:49]: specify nydus blob compression algorithm, possible values: `none`, `lz4_block`, `gzip`, `zstd`, default is `lz4_block`?

File: misc/config/config.yaml.nydus.tmpl [hunk 2]
```diff
@@ -46,6 +46,8 @@ converter:
...
+      # rafs_compressor: zstd
```
Comment (imeoer) [inline misc/config/config.yaml.nydus.tmpl:50]: Why not default value?

**Comment by imeoer** [comment]
Please fix the broken CI.

File: misc/config/config.yaml.nydus.tmpl (hunk 2 above)
Comment (hsiangkao) [reply misc/config/config.yaml.nydus.tmpl:50]: it depends on the actual scenarios, I'm not sure if it's optimal for internal network.

File: misc/config/config.yaml.nydus.tmpl (hunk 1 above)
Comment (hsiangkao) [reply misc/config/config.yaml.nydus.tmpl]: done.

**Comment by hsiangkao** [comment]
> Please fix the broken CI.

done.

**Comment by imeoer** [comment]
Other LGTM, thanks for the contribution!

**Pull request merged**